python/tvm/topi/adreno/conv2d_nchw.py

```diff
@@ -268,7 +268,11 @@ def schedule_conv2d_NCHWc_KCRSk(cfg, s, output):
     cfg.define_split("tile_rx", rx, num_outputs=2)
     cfg.define_knob("auto_unroll_max_step", [0, 512, 1500])
     cfg.define_knob("unroll_explicit", [0, 1])
-
+    cfg.multi_filter(
+        filter=lambda entity: 32
```

Comment:
Added into python\tvm\topi\adreno\conv2d_nhwc.py

It might make sense to add it for the next implementations, too?
python\tvm\topi\adreno\depthwise_conv2d_nchw.py
python\tvm\topi\adreno\depthwise_conv2d_nhwc.py

Reply:
good point

Reply:
Added.